docs/future/compatibility.md

```diff
@@ -0,0 +1,191 @@
+# Enzyme Adapter & Compatibility Proposal
+
+
+## Motivation
+
+This proposal is attempting to address a hand full of pain points that Enzyme has been
+suspect to for quite a while.  This proposal has resulted mostly [#715](https://github.com/airbnb/enzyme/issues/715),
```

Comment:
`s/suspect/subject` (also, no need to put two spaces after a period unless on a typewriter ;-) )